### ISSUE TITLE
View Response In HTML

### ISSUE THREAD

**Issue by AustinMCrane** [mmattozzi/cocoa-rest-client#70]
This is a great development tool as is but would be greatly improved with the capability to see http response body in html form. Rails gives some ridiculously hard to follow html code for many errors which makes debugging a pain. I might be able to write a embeddable web view into the app if the community would also benefit from this feature.

**Comment by mmattozzi** [comment]
I added a menu option a while ago for "View response in browser" ... although when I just clicked it on my computer, I got an error message about not being able to discover the default browser, so I may need to look into that. Does this work for you? It also probably wouldn't be too hard to add a WebView to the response pane, but I don't want to try to rebuild an entire web browser here. :)

**Comment by AustinMCrane** [comment]
My apologies, where is this button at? If it is already available than that is awesome!

**Comment by mmattozzi** [comment]
It's in the File menu, `View Response in Browser`
